src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -49,9 +27,8 @@ else
     <PackageReference Include=\"Microsoft.Portable.Targets\" Version=\"$__PORTABLETARGETS_VERSION\" />
   </ItemGroup>
 </Project>"
-    __PROJECT_EXTENSION=csproj
-    __PUBLISH_TFM=netcoreapp2.0
-fi
+__PROJECT_EXTENSION=csproj
```

Comment:
I don't think `__PROJECT_EXTENSION` is used anymore after this change, so can be removed.